pkg/clusterversion/cockroach_versions.go

```diff
@@ -104,7 +104,9 @@ type Key int
 // When introducing a version constant, you'll want to:
 //   (1) Add it at the end of this block. For versions introduced during and
 //       after the 21.1 release, Internal versions must be even-numbered. The
-//       odd versions are used for internal book-keeping.
+//       odd versions are used for internal book-keeping. The Internal version
+//       should be the previous Internal version for the same minor release plus
```

Comment:
Thanks for adding this comment! Internal versions are independent from minor releases, a.k.a. patch releases, so this might be a bit misleading. We don't add internal versions in minor releases; internal releases should be thought of as the upgrade steps from one major release to another. If you find this confusing, you're not alone.